hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/FederationPage.java

```diff
@@ -47,10 +48,10 @@ protected Class<? extends SubView> content() {
 
   private String rmsTableInit() {
     StringBuilder b = tableInit().append(", aoColumnDefs: [");
-    b.append("{'bSearchable': false, 'aTargets': [ 7 ]}")
+    b.append("{'bSearchable': false, 'aTargets': [ 2 ]}")
```

Comment:
As we are at it, can we make this numbers variables with names?

Reply:
I'll fix it.